### PR TITLE
Faq common sc text

### PR DESCRIPTION
**Pull request by nomadscientist**
I'm making the tutorials and pages more consistent
With turning common text into FAQ and fixing some formatting issues

### CODE REVIEW

File: topics/single-cell/index.md [hunk 1]
```diff
@@ -13,6 +13,6 @@ Check out workflows shared by users like you!
 
 If you want to help us behind the scenes, from testing workflows and tutorials to building tools, join our Galaxy Single Cell Community of Practice!
 
- - <i class="fa fa-hand-o-right" aria-hidden="true"></i> [Community of Practice](https://galaxyproject.org/projects/singlecell/)
- - <i class="fa fa-comments-o" aria-hidden="true"></i> [Matrix Chat Forum](https://matrix.to/#/#usegalaxy-eu_single-cell-workflows:gitter.im)
+ - <i class="fa fa-hand-o-right" aria-hidden="true"> </i> [Community of Practice](https://galaxyproject.org/projects/singlecell/)
+ - <i class="fa fa-comments-o" aria-hidden="true"> </i> [Matrix Chat Forum](https://matrix.to/#/#usegalaxy-eu_single-cell-workflows:gitter.im)
  - <i class="fa fa-envelope" aria-hidden="true"> </i> [Mailing List](https://lists.galaxyproject.org/lists/single-cell-cop.lists.galaxyproject.org/)
```
Comment (hexylena) [inline topics/single-cell/index.md:18]: `{% icon .. %}` should function here, let's add support for whatever icons you need to `_config.yml` (just give them nice human names) to make it a bit more readable for you and others?

```suggestion
 - <i class="fa fa-hand-o-right" aria-hidden="true"> </i> [Community of Practice](https://galaxyproject.org/projects/singlecell/)
 - {% icon feedback %} [Matrix Chat Forum](https://matrix.to/#/#usegalaxy-eu_single-cell-workflows:gitter.im)
 - <i class="fa fa-envelope" aria-hidden="true"> </i> [Mailing List](https://lists.galaxyproject.org/lists/single-cell-cop.lists.galaxyproject.org/)
```

**Comment by nomadscientist** [comment]
Also addressed this issue: https://github.com/galaxyproject/training-material/issues/2799

File: topics/single-cell/index.md (hunk 1 above)
Comment (nomadscientist) [reply topics/single-cell/index.md:18]: like this?   point-right: fa fa-hand-o-right

Comment (hexylena) [reply topics/single-cell/index.md]: yep! fas / far are 'solid' and 'regular', whichever one is fitting better + free (if you check the fontawesome website/search)

**Comment by nomadscientist** [comment]
removed the FPE fix to another branch since I have to deal with workflow testing...

**Comment by nomadscientist** [comment]
Yes I think that's an awesome idea! Idk how to make the subsections without examples yet but I think that's a great call
It goes hand-in-hand with us needing to change the help forum to switch from scrna as the only option to single cell as well

**Comment by pavanvidem** [comment]
Then we should also merge all the markdowns into 3 files to fit the sections if that's possible.

**Pull request merged**